### PR TITLE
100% coverage

### PR DESCRIPTION
**Pull request by homburg**

### CODE REVIEW

**Comment by homburg** [comment]
https://travis-ci.org/homburg/rally/jobs/57689672

File: rally_test.go [hunk 1]
```diff
@@ -16,6 +16,20 @@ import (
 
 const token = "token"
 
+func TestGetError(t *testing.T) {
```
Comment (ofmadsen) [inline rally_test.go]: Could this method be given a more explanatory name; e.g. `TestFailReadResponseBody`? Also it should be moved further below in the file, after `TestFailStatusCodeNotOK`. The first tests in the file is the sunshine scenarios.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+language: go
+go:
+  - 1.2
+  - 1.3
+  - 1.4
+script: go test -v -cover ./...
```
Comment (ofmadsen) [inline .travis.yml:6]: Good idea! I'll be able to use the pretty emblems in the README!

**Comment by ofmadsen** [comment]
100% is awesome! Thank you - I had no idea how to test that

File: .travis.yml (hunk 2 above)
Comment (homburg) [inline .travis.yml:6]: Exactly :-)

File: rally_test.go (hunk 1 above)
Comment (homburg) [inline rally_test.go]: Done :-)

**Pull request merged**